### PR TITLE
Add static asset tooling

### PR DESCRIPTION
**Pull request by sarahquigley**
Tooling for compiling and watching static assets, running front-end unit tests, and serving the site (via `manage.py runserver`) concurrently.

### CODE REVIEW

**Comment by sarahquigley** [comment]
@mstewart I would appreciate you reviewing this. Thank you! :tanabata_tree:

File: django_basic_blog/management/commands/gruntserver.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from django.contrib.staticfiles.management.commands.runserver import Command as StaticfilesRunserverCommand
+
+class Command(StaticfilesRunserverCommand):
+
+  def inner_run(self, *args, **options):
+    self.start_grunt()
+    return super(Command, self).inner_run(*args, **options)
+
+  def start_grunt(self):
+    self.stdout.write('>>> Starting grunt')
```
Comment (mstewart) [inline django_basic_blog/management/commands/gruntserver.py]: You probably want a trailing newline on all these `write` calls, for legibility.  Or you can just `print` your message, which will auto-add newlines.

**Comment by mstewart** [comment]
Looks great! :+1: 

I notice all the scss and coffee files are under the `static/` tree, too. Do these get served as static assets by Django, alongside the compiled js/css?

**Comment by sarahquigley** [comment]
:lemon:

**Pull request merged**